cmd/serve/main.go

```diff
@@ -42,25 +45,44 @@ func main() {
 }
 
 func run() error {
-	environment := di.GetEnvironment()
+	config := config.ProvideConfig()
```

Comment:
Intellij will warn about variable name will collide with package name. Shall we agree on either cfg or just c? Whatever you prefer :-)

Reply:
ah, good catch! yes, I lean towards `cfg`. I think `c` is often used for a `Client` like struct for example in a receiver.